### PR TITLE
feat: eslint v9 & flat config setup

### PR DESCRIPTION
**Pull request by ixahmedxi**

### CODE REVIEW

File: apps/marketing/eslint.config.js
Comment (imCorfitz) [inline apps/marketing/eslint.config.js:1]: This is lovely 😍

File: packages/ui/src/primitives/navigation-menu/index.tsx
Comment (imCorfitz) [inline packages/ui/src/primitives/navigation-menu/index.tsx:1]: Have all of these tailwind updates been automatically updated by `lint --fix`?

File: packages/ui/src/primitives/radio-group/index.tsx [hunk 1]
```diff
@@ -29,13 +29,13 @@ const RadioGroupItem = React.forwardRef<
     <RadioGroupPrimitive.Item
       ref={ref}
       className={cn(
-        'aspect-square h-4 w-4 rounded-full border border-primary text-primary shadow focus:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50',
+        'aspect-[1_/_1] size-4 rounded-full border border-primary text-primary shadow focus:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50',
```
Comment (imCorfitz) [inline packages/ui/src/primitives/radio-group/index.tsx:32]: odd that `aspect-square` was changed to `aspect-[1_/_1]` in this case 🤔

**Review by imCorfitz — APPROVED** [review]
lgtm

File: packages/ui/src/primitives/navigation-menu/index.tsx
Comment (ixahmedxi) [reply packages/ui/src/primitives/navigation-menu/index.tsx:1]: yeah that is true

**Pull request merged**